rocketmq-connect/rocketmq-connect-runtime/src/main/java/org/apache/rocketmq/connect/runtime/connectorwrapper/WorkerSourceTask.java

```diff
@@ -136,6 +136,7 @@ public KeyValue configs() {
                 try {
                     Collection<SourceDataEntry> toSendEntries = sourceTask.poll();
                     if (null != toSendEntries && toSendEntries.size() > 0) {
+                        log.info("send records");
```

Comment:
I left some very bad log conventions in the runtime  and I apologize for that, but we need to try to make log messages clear ~ This might be a log added during debug process so we might not need it ~ I'll probably refactor the logs in externals (as some of them looks messy, with some needed logs missing) it would be great if you're willing to help :)

Reply:
Updated